### PR TITLE
prepare structure for 0.19

### PR DESCRIPTION
**Pull request by gyzerok**
In 0.19 package projects will have to contain their modules inside `src` directory. This PR prepares library for the change.

### CODE REVIEW

File: tests/VerifyExamples/Dict/Verify/NotEmpty0.elm [hunk 1]
```diff
@@ -0,0 +1,28 @@
+module VerifyExamples.Dict.Verify.NotEmpty0 exposing (..)
```
Comment (stoeffel) [inline tests/VerifyExamples/Dict/Verify/NotEmpty0.elm]: could you add `tests/VerifyExamples` to `.gitignore

Comment (gyzerok) [reply tests/VerifyExamples/Dict/Verify/NotEmpty0.elm]: Oh, sorry, my bad. Will fix.

Comment (stoeffel) [reply tests/VerifyExamples/Dict/Verify/NotEmpty0.elm]: no worries. it was in there as `test/doc` (which was the folder in an older version of `elm-verified-examples`.

**Comment by gyzerok** [comment]
@stoeffel done

**Pull request merged**